regnety/models/model.py

```diff
@@ -0,0 +1,69 @@
+import tensorflow as tf
+
+from regnety.regnety.config import get_model_config, ALLOWED_FLOPS
+from regnety.regnety.models.blocks import PreStem, Stem, Stage, Head
+
+class RegNetY(tf.keras.Model):
+    """
+    RegNetY model class. Subclassed from tf.keras.Model. Instantiates a randomly
+    initialised model (to be changed after training). User must provide `flops`
+    argument to model.
+
+    Args:
+        flops: Flops of the model eg. "400MF"
+    """
+
+    def __init__(self, flops:str = ""):
+        super(RegNetY, self).__init__()
+
+        if flops not in ALLOWED_FLOPS:
+            raise ValueError("`flops` must be one of " + str(ALLOWED_FLOPS))
+        
+        self.flops = flops
+        self.config = get_model_config(self.flops)
+        self.model = self._get_model_with_config(self.config)
+
+    def build(self, input_shape):
+        self.model.build(input_shape)
+
+    def call(self, inputs):
+        self.model.call(inputs)
+    
+    def get_model(self):
+        """Returns sequential model constructed in this class"""
+        return self.model
+    
+
+    def _get_model_with_config(self, config):
```

Comment:
Pretty cool to see what one can achieve even with `Sequential`. Great work framing the blocks in such a manner.